scripts/common/common.sh

```diff
@@ -219,6 +219,17 @@ function has_default_namespace() {
     kubectl get ns | grep -q '^default' 2>/dev/null
 }
 
+function wait_for_kube_apiserver(){
+        if ! spinner_until 120 get_kube_apiserver_succeeds ; then
+        # this should exit script on non-zero exit code and print error message
+        kubectl get pod -n kube-system -l component=kube-apiserver 1>/dev/null
+    fi
+}
+
+function get_kube_apiserver_succeeds() {
+    kubectl get pod -n kube-system -l component=kube-apiserver >/dev/null 2>&1
+}
```

Comment:
is this the same as kubeadm_api_is_healthy?

https://github.com/replicatedhq/kURL/blob/2f3e46c9c4618f539f50fd99d9dc589034901368/scripts/distro/kubeadm/distro.sh#L185-L187

Reply:
yes, this one is better as it will check for api-server to be in running state